src/main/java/com/damoim/restapi/seminar/entity/Seminar.java

```diff
@@ -0,0 +1,44 @@
+package com.damoim.restapi.seminar.entity;
+
+import lombok.*;
+
+import javax.persistence.*;
+import javax.validation.constraints.NotBlank;
+import java.time.LocalDateTime;
+
+/**
+ * @author gjsung.Go
+ * @since 2012.02.20
+ */
+@Entity
+@Getter
+@Setter
+@Builder
+@NoArgsConstructor
+@AllArgsConstructor
+public class Seminar extends BaseEntity {
+    @Id @GeneratedValue(strategy = GenerationType.IDENTITY)
+    @Column(name = "seminar_id")
```

Comment:
인조키의 컬럼 네임을 seminar_id로 만든 의도가 궁금합니다!

Reply:
@Gogisung 주석 since 날짜가 잘못 되었습니다 : ) 
언더바 사용은 지양하는 것이 좋아보입니다.
주석은 보통 식별 가능하거나 의미가 분명한 필드일 경우 전부 작성하는 것이 불필요하게 느껴집니다. 
멀티 라인 주석 작성시 간략한 기능 명세에 대한 description 정도가 어떨까요?

Reply:
@dodo4513 JPA 연관관계를 설정 하려고 네임을 만들었으나 사용하지 않아 지우겠습니다